packages/lit-auth-client/src/lib/providers/EthWalletProvider.ts

```diff
@@ -134,7 +134,7 @@ export default class EthWalletProvider extends BaseProvider {
     }
 
     address = ethers.utils.getAddress(address);
-
+    const nonce = await litNodeClient.getLatestBlockhash();
```

Comment:
We might still need the nonce in either branch of the `if` so I moved it outside of the condition.


However, thinking about this change a bit more i do wonder if we should just use `latestBlockhash` internally and not use `getLatestBlockhash` since we do network syncs. 
@Ansonhkg @gilbert  what do you think? I was away when these changes where discussed. I am worried if we start calling `getLatestBlockhash` internally we are only furthering the indeterminant I/O problems.

Reply:
Our next iteration is to _remove_ the polling that is populating the `latestBlockhash` property and make it `private`, so consumers always call the async get -- this lets us fetch it on demand and not deal with caching it.  The idea being that we would then start fetching it from somewhere other than handshaking with the nodes -- the current implementation is kind of a 'stepping stone' -- that's why I was saying the other day that we probably should've marked `latestBlockhash` as `private` in our v5.0.0 release :(

Reply:
agreed that we should no longer recommend outside callers to the SDK to use `latestBlockhash` but for internal usage i think we should still use it over `getLatestBlockhash` since it will cause network overhead. In the next release which is breaking we can include making `latestBlockhash` private.